eden/scm/edenscm/ext/github/pull_request_body.py

```diff
@@ -16,6 +16,7 @@ def create_pull_request_title_and_body(
     pr_numbers_and_num_commits: List[Tuple[int, int]],
     pr_numbers_index: int,
     repository: Repository,
+    skip_reviewstack_tag: False,
```

Comment:
Alternatively we could allow callers to provide a template but that seems much more complex. Thoughts?

Reply:
Sorry for the slow review. I think template is more flexible. But this looks okay too.